plugin/storage/grpc/config/config.go

```diff
@@ -67,11 +67,30 @@ func (c *Configuration) Build() (shared.StoragePlugin, error) {
 	if !ok {
 		return nil, fmt.Errorf("unexpected type for plugin \"%s\"", shared.StoragePluginIdentifier)
 	}
+	archiveStoragePlugin, ok := raw.(shared.ArchiveStoragePlugin)
+	if !ok {
+		return nil, fmt.Errorf("unexpected type for plugin \"%s\"", shared.StoragePluginIdentifier)
```

Comment:
We could help debugging by including the type of `raw` in these error messages.  Under normal circumstances this wouldn't fail, but if it did this would be a good clue to finding out why.  Also differentiating the error messages would also be nice `unable to cast %T to shared.ArchiveStoragePlugin` or something